src/Common/Constants.ts

```diff
@@ -145,6 +145,9 @@ export class Queries {
   public static QueryEditorMinHeightRatio: number = 0.1;
   public static QueryEditorMaxHeightRatio: number = 0.4;
   public static readonly DefaultMaxDegreeOfParallelism = 6;
+  public static readonly DefaultRetryAttempts = 9;
+  public static readonly DefaultRetryIntervalInMs = 0;
```

Comment:
The tooltip for this says that default is 5000 ms ?

Reply:
Sorry, saw the default was 0 so changed it. I have corrected the tooltip.